commit bb2c205cd3a0e8d9c7597b67eb0be793d25854b7
Author: Marek Szyprowski <m.szyprowski@samsung.com>
Date:   Mon Feb 25 19:42:52 2019 +0100

    ARM: dts: exynos: Fix max voltage for buck8 regulator on Odroid XU3/XU4
    
    commit a3238924a820c1d7c977b632b769f3b5690cba09 upstream.
    
    The maximum voltage value for buck8 regulator on Odroid XU3/XU4 boards is
    set too low. Increase it to the 2000mV as specified on the board schematic.
    So far the board worked fine, because of the bug in the PMIC driver, which
    used incorrect step value for that regulator. It interpreted the voltage
    value set by the bootloader as 1225mV and kept it unchanged. The regulator
    driver has been however fixed recently in the commit 56b5d4ea778c
    ("regulator: s2mps11: Fix steps for buck7, buck8 and LDO35"), what results
    in reading the proper buck8 value and forcing it to 1500mV on boot. This
    is not enough for proper board operation and results in eMMC errors during
    heavy IO traffic. Increasing maximum voltage value for buck8 restores
    original driver behavior and fixes eMMC issues.
    
    Signed-off-by: Marek Szyprowski <m.szyprowski@samsung.com>
    Fixes: 86a2d2ac5e5d ("ARM: dts: Add dts file for Odroid XU3 board")
    Fixes: 56b5d4ea778c ("regulator: s2mps11: Fix steps for buck7, buck8 and LDO35")
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Krzysztof Kozlowski <krzk@kernel.org>
    Signed-off-by: Arnd Bergmann <arnd@arndb.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm/boot/dts/exynos5422-odroid-core.dtsi b/arch/arm/boot/dts/exynos5422-odroid-core.dtsi
index 2f4f40882dab..27214e6ebe4f 100644
--- a/arch/arm/boot/dts/exynos5422-odroid-core.dtsi
+++ b/arch/arm/boot/dts/exynos5422-odroid-core.dtsi
@@ -334,7 +334,7 @@
 			buck8_reg: BUCK8 {
 				regulator-name = "vdd_1.8v_ldo";
 				regulator-min-microvolt = <800000>;
-				regulator-max-microvolt = <1500000>;
+				regulator-max-microvolt = <2000000>;
 				regulator-always-on;
 				regulator-boot-on;
 			};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1131_bb2c205cd3a0e8d9c7597b67eb0be793d25854b7.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1131_bb2c205cd3a0e8d9c7597b67eb0be793d25854b7.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
